### PR TITLE
add:`g4music`

### PR DESCRIPTION
**Pull request by soumyatheman**
G4music added with proper deps

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Run the pre-commit hooks

**Comment by soumyatheman** [comment]
> Run the pre-commit hooks

I ran it, The packages are not available in the debian repos, so they failed

**Comment by soumyatheman** [comment]
> Run the pre-commit hooks

Just to recheck How to run pre commit hooks?

**Comment by wizard-28** [comment]
> > Run the pre-commit hooks
> 
> Just to recheck How to run pre commit hooks?

Install `pre-commit`, `shellcheck` and `shfmt` from here: https://github.com/pacstall/pacstall-programs#how-to-setup-the-environment-for-pacscript-development

Then run `pre-commit install`, then run `pre-commit run --files packagelist packages/g4music/g4music.pacscript`

**Comment by soumyatheman** [comment]
I did it correctly and all is fine
````
trim trailing whitespace.............................(no files to check)Skipped
fix end of files.....................................(no files to check)Skipped
check yaml...........................................(no files to check)Skipped
check for added large files..........................(no files to check)Skipped
file contents sorter.................................(no files to check)Skipped
mixed line ending....................................(no files to check)Skipped
Test shell scripts with shellcheck...................(no files to check)Skipped
Check shell style with shfmt.........................(no files to check)Skipped

````

**Comment by wizard-28** [comment]
> I did it correctly and all is fine
> 
> ```
> trim trailing whitespace.............................(no files to check)Skipped
> fix end of files.....................................(no files to check)Skipped
> check yaml...........................................(no files to check)Skipped
> check for added large files..........................(no files to check)Skipped
> file contents sorter.................................(no files to check)Skipped
> mixed line ending....................................(no files to check)Skipped
> Test shell scripts with shellcheck...................(no files to check)Skipped
> Check shell style with shfmt.........................(no files to check)Skipped
> ```

No, run the command from the project root

**Comment by soumyatheman** [comment]
Yess I did that

**Comment by soumyatheman** [comment]
`````
 soumyaxubuntu  soumyas-HP-Laptop-15-bs1xx  ../pacstall-programs  master ◔  pre-commit run --files packagelist packages/g4music/g4music.pacscript
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check yaml...........................................(no files to check)Skipped
check for added large files..............................................Passed
file contents sorter.....................................................Passed
mixed line ending........................................................Passed
Test shell scripts with shellcheck.......................................Passed
Check shell style with shfmt.............................................Failed
- hook id: shfmt
- files were modified by this hook

[RUN] shfmt -w -i 2 -bn -ci -sr -kp -s packages/g4music/g4music.pacscript
``````

**Comment by wizard-28** [comment]
Now amend your last commit: `git commit --amend --no-edit`, then `git push --force-with-lease`

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Remove `packages/newsflash/.newsflash.pacscript.swp`, I don't know why you committed that?

**Comment by soumyatheman** [comment]
I am commiting it in another branch, how did that came in!!!!

**Comment by wizard-28** [comment]
> I am seriously freaking out now!!! What's the issue, the last time all the tests passed and u said I mistakenly added a file from another branch, now I fixed that and again it's not passing all the tests. Why are these heppening!!!!!

This isn't magic, just check the CI runs... https://github.com/pacstall/pacstall-programs/actions/runs/3665890416/jobs/6197264800

Debian doesn't have `libgstreamer-plugins-good1.0-0`, so the installation fails

**Comment by soumyatheman** [comment]
Yess I know that. But last timw somehow it happened. So will you merge this?Sent using Zoho Mail ---- On Sun, 11 Dec 2022 02:26:37 +0530  ***@***.***  wrote ----

I am seriously freaking out now!!! What's the issue, the last time all the tests passed and u said I mistakenly added a file from another branch, now I fixed that and again it's not passing all the tests. Why are these heppening!!!!!

This isn't magic, just check the CI runs... https://github.com/pacstall/pacstall-programs/actions/runs/3665890416/jobs/6197264800
Debian doesn't have libgstreamer-plugins-good1.0-0, so the installation fails

**Comment by wizard-28** [comment]
> Yess I know that. But last timw somehow it happened. So will you merge this?Sent using Zoho Mail ---- On Sun, 11 Dec 2022 02:26:37 +0530 ***@***.*** wrote ---- I am seriously freaking out now!!! What's the issue, the last time all the tests passed and u said I mistakenly added a file from another branch, now I fixed that and again it's not passing all the tests. Why are these heppening!!!!! This isn't magic, just check the CI runs... [pacstall/pacstall-programs/actions/runs/3665890416/jobs/6197264800](https://github.com/pacstall/pacstall-programs/actions/runs/3665890416/jobs/6197264800) Debian doesn't have libgstreamer-plugins-good1.0-0, so the installation fails —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were assigned.Message ID: ***@***.***>

The CI has never passed in all the PRs you've made for this pacscript.

Check if `libgstreamer-plugins-base1.0-0` works instead. Otherwise, we'd need to stall this PR until Pacstall 3.1.0 is released with the [`incompatible` array](https://github.com/pacstall/pacstall/pull/717) feature.

**Comment by soumyatheman** [comment]
Nah it won't, it needs gstreamer-tag-1.0-0Sent using Zoho Mail ---- On Sun, 11 Dec 2022 02:36:16 +0530  ***@***.***  wrote ----



Yess I know that. But last timw somehow it happened. So will you merge this?Sent using Zoho Mail ---- On Sun, 11 Dec 2022 02:26:37 +0530 @.*** wrote ---- I am seriously freaking out now!!! What's the issue, the last time all the tests passed and u said I mistakenly added a file from another branch, now I fixed that and again it's not passing all the tests. Why are these heppening!!!!! This isn't magic, just check the CI runs... pacstall/pacstall-programs/actions/runs/3665890416/jobs/6197264800 Debian doesn't have libgstreamer-plugins-good1.0-0, so the installation fails —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were assigned.Message ID: @.***>



The CI has never passed in all the PRs you've made for this pacscript.

Check if libgstreamer-plugins-base1.0-0 works instead. Otherwise, we'd need to stall this PR until Pacstall 3.1.0 is released with the incompatible array feature.

File: packages/webcord-deb/webcord-deb.pacscript [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 gives="webcord"
-version="3.10.1"
+version="4.0.0"
 replace="${gives}"
 url="https://github.com/SpacingBat3/WebCord/releases/download/v${version}/webcord_${version}_amd64.deb"
 repology=("project: webcord")
 description="A Discord and Fosscord client implemented directly without Discord API."
-hash="78901c1f0eecee91c7922eb163540ef136b6779b0f1eb10c54676a4013b3c9f3"
+hash="0e5f2070a060f1e10b0a427a11347ce825b113428ab1ad120d76cbe73a3b92b6"
 maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
```
Comment (Henryws) [inline packages/webcord-deb/webcord-deb.pacscript:9]: Why? This is not `g4music`. Remove it.

Comment (wizard-28) [reply packages/webcord-deb/webcord-deb.pacscript:9]: Remove your last commit, don't make a new commit to remove it.

**Comment by Mar0xy** [comment]
> Debian doesn't have `libgstreamer-plugins-good1.0-0`,

Debian does have plugins-good just this is another case of it being called different: `gstreamer1.0-plugins-good`





> Nah it won't, it needs gstreamer-tag-1.0-0

gstreamer-tag is already included in `libgstreamer-plugins-base1.0-dev` on debian

**Pull request closed without merging**